src/components/Editor/Editor.tsx

```diff
@@ -109,6 +111,15 @@ const Editor: FC<Props> = ({
     )
   );
 
+  useEffect(() => {
```

Comment:
Could You elaborate why this `useEffect` is required here ? The widget customization input settings are serialized during first render.